src/web/routes/application/flow-control/state-machine/selectors/selectors.js

```diff
@@ -1,10 +1,10 @@
-const { CHECK_ANSWERS_URL, TERMS_AND_CONDITIONS_URL } = require('../../../paths')
+const { CHECK_ANSWERS_URL, TERMS_AND_CONDITIONS_URL, prefixPath } = require('../../../paths')
 const { isNextPathNavigable } = require('../predicates')
 const { getNextForStep } = require('./get-next-for-step')
 
 const getStepForPath = (path, steps) => steps.find(step => step.path === path)
 
-const getNextInReviewPath = (req) => req.path === CHECK_ANSWERS_URL ? TERMS_AND_CONDITIONS_URL : CHECK_ANSWERS_URL
+const getNextInReviewPath = (req, prefix) => req.path === prefixPath(prefix, CHECK_ANSWERS_URL) ? prefixPath(prefix, TERMS_AND_CONDITIONS_URL) : prefixPath(prefix, CHECK_ANSWERS_URL)
```

Comment:
Is it worth pulling `prefixPath(prefix, CHECK_ANSWERS_URL)` into a `const`? It's used twice in this ternary.